pkgs/servers/web-apps/davical/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, gettext, fetchFromGitLab }:
+
+let
+  awl = import ./awl.nix { inherit stdenv lib gettext fetchFromGitLab; };
```

Comment:
It is better to expose awl in all-packages.nix so that the end user can override it in an overlay.